certbot/certbot/_internal/auth_handler.py

```diff
@@ -87,9 +87,34 @@ def handle_authorizations(self, orderr: messages.OrderResource,
 
                 # If debug is on, wait for user input before starting the verification process.
                 if config.debug_challenges:
+                    msg = []
+                    if config.verbose_count > 0:
+                        http01_achalls = {}
+                        dns01_achalls = {}
+                        for achall in achalls:
+                            if isinstance(achall.chall, challenges.HTTP01):
+                                http01_achalls[achall.chall.uri(achall.domain)] = (
+                                    achall.validation(achall.account_key) + "\n"
+                                )
+                            if isinstance(achall.chall, challenges.DNS01):
+                                dns01_achalls[achall.validation_domain_name(achall.domain)] = (
+                                    achall.validation(achall.account_key) + "\n"
+                                )
+                        if http01_achalls:
+                            msg.append("\nThe following URLs should be accessible from the world "
+                                       "wide web and return the value mentioned:\n")
```

Comment:
I appreciate the flavor here 😆 .

It's not a big deal, but do you think the message will get across to more people if we call it the "internet"?

Reply:
Not sure if people wouldn't understand "world wide web", but it's probably good to err on the side of caution, thanks :slightly_smiling_face: